.gitignore

```diff
@@ -16,7 +16,6 @@ commcare-odk-test/gen/*
 *.swp
 *.iml
 *.db

```

Comment:
I had to remove `*.properties` and `**/.settings/` from .gitignore in order for these 2 files within the `unit-tests` directory to be properly recognized as having moved: https://github.com/dimagi/commcare-android/blob/master/unit-tests/.settings/org.eclipse.jdt.core.prefs and https://github.com/dimagi/commcare-android/blob/master/unit-tests/resources/robolectric.properties. I'm not sure how they were previously being recognized with these lines in there.